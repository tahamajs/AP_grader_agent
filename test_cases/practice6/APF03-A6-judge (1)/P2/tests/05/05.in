POST signup ? username "low_mist" password "meow"
POST logout ?
POST signup ? username "cool_man" password "1234"
POST logout ?
POST signup ? username "amir_akbar" password "1382"
POST increase_budget ? amount 600
GET reserves ? restaurant_name "sib"
POST reserve ? restaurant_name "sib" table_id "2" start_time "16" end_time "19" foods "burger,pitza"
GET reserves ? restaurant_name "sib" reserve_id "1"
POST logout ?
POST login ? username "cool_man" password "1234"
POST logout ?
POST login ? username "low_mist" password "meow"
POST increase_budget ? amount 500
POST reserve ? restaurant_name "sib" table_id "2" start_time "19" end_time "21" foods "pitza,pitza,burger"
GET reserves ? reserve_id "1"
GET reserves ? restaurant_name "sib" reserve_id "1"
GET reserves ? restaurant_name "sib" reserve_id "335"
POST logout ?
POST login ? username "cool_man" password "1234"
POST increase_budget ? amount 600
POST reserve ? restaurant_name "hani" table_id "1" start_time "18" end_time "21" foods "cholo morgh,joojeh,joojeh"
GET reserves ? restaurant_name "hani" reserve_id "1"
POST logout ?
POST login ? username "amir_akbar" password "1382"
POST reserve ? restaurant_name "burger land" table_id "1" start_time "14" end_time "15"
GET reserves ?
GET reserves ? restaurant_name "sib" reserve_id "1"
POST logout ?