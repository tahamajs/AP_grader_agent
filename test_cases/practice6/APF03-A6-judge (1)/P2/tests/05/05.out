OK
OK
OK
OK
OK
OK
Empty
Reserve ID: 1
Table 2 for 16 to 19 in sib
Original Price: 560
Order Amount Discount: 0
Total ItemSpecific Discount: 275
First Order Discount: 0
Total Discount: 275
Total Price: 285
1: sib 2 16-19 burger(1) pitza(1) 560 285
OK
OK
OK
OK
OK
Reserve ID: 2
Table 2 for 19 to 21 in sib
Original Price: 910
Order Amount Discount: 138
Total ItemSpecific Discount: 450
First Order Discount: 0
Total Discount: 588
Total Price: 322
Bad Request
Permission Denied
Not Found
OK
OK
OK
Reserve ID: 1
Table 1 for 18 to 21 in hani
Original Price: 760
Order Amount Discount: 204
Total ItemSpecific Discount: 40
First Order Discount: 40
Total Discount: 284
Total Price: 476
1: hani 1 18-21 cholo morgh(1) joojeh(2) 760 476
OK
OK
Reserve ID: 1
Table 1 for 14 to 15 in burger land
Original Price: 0
Order Amount Discount: 0
Total ItemSpecific Discount: 0
First Order Discount: 0
Total Discount: 0
Total Price: 0
1: burger land 1 14-15 0 0
1: sib 2 16-19 burger(1) pitza(1) 560 285
1: sib 2 16-19 burger(1) pitza(1) 560 285
OK